.github/workflows/windows_unit_tests.yml

```
@@ -38,9 +38,7 @@ jobs:
         shell: cmd
         run: |
           python -m pip install --upgrade pip
-          pip install flake8 pytest pytest-cov
-          pip install psutil joblib
-          pip install --verbose .[gui,opt]
+          pip install --verbose .[opt,parallel,test,gui]
```

```suggestion
          pip install --verbose .[opt,parallel, test, gui]
```
Just to match the above (there's a list below this one that could use spaces too)

---

oh hah. Apparently windows cmd prompt doesn't like the spaces. I've converted it back to no spaces!